packages/@react-spectrum/dnd/docs/useDragHooks.mdx

```diff
@@ -0,0 +1,166 @@
+{/* Copyright 2022 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License. */}
+
+import {Layout} from '@react-spectrum/docs';
+export default Layout;
+
+import ChevronRight from '@spectrum-icons/workflow/ChevronRight';
+import docs from 'docs:@react-spectrum/dnd';
+import listviewDocs from 'docs:@react-spectrum/list';
+import {FunctionAPI, PageDescription} from '@react-spectrum/docs';
+import {Keyboard} from '@react-spectrum/text';
+
+---
+category: Concepts
+keywords: [drag and drop, dnd]
+---
+
```

Comment:
Open to opinions on this page's content. I figured I'd say out of the nitty gritty of the aria dnd hooks for now

Reply:
A few examples that might fit well when everything is ready: 
- Drag items out of list (i.e. existing story)
- Drag items within list (Reorder)
- Drag non-folder items into folder items
- Drag items between lists (Kanban board)

Reply:
And may be good to hear any more common use cases seen in the wild (cc: @matthewdeutsch)